packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -23,6 +23,12 @@ module.exports = {
     ['meta', { 'http-equiv': 'XA-UA-Compatible', content: 'IE=edge'}],
     ['meta', { name: 'viewport', content: 'width=device-width, initial-scale=1.0'}],
 
+    /*
+     * Okta sign-in widget
+     */
+    ['link', { href: 'https://global.oktacdn.com/okta-signin-widget/5.2.0/css/okta-sign-in.min.css', type: 'text/css', rel: 'stylesheet'}],
```

Comment:
- Create a variable for the version number to allow easy upgrade?
- Is the comment in the editor auto generated?. Are we able to update that as well?

`// Uses okta-signin-widget version 5.2.0`

Reply:
I just found this widget version script that exists for updating the documentation automatically so I'm going to try to leverage it here

Reply:
Updated it -- this pulls from the same version that is shown on our docs pages now and it works really well